.tern-project

```diff
@@ -0,0 +1,16 @@
+{
```

Comment:
[ternjs](http://ternjs.net/) enables autocomplete and code analyzes for javascript files.
Many editors have plugins that work with it, this file is the project configuration for ternjs.